airgun/entities/filter.py

```diff
@@ -29,6 +29,24 @@ def read(self, role_name, entity_name, widget_names=None):
             self, 'Edit', role_name=role_name, entity_name=entity_name)
         return view.read(widget_names=widget_names)
 
+    def read_all(self, role_name):
```

Comment:
Not sure about this name... in test code, we would have `session.filter.read_all(role_name)`. If I saw that, I think I would have trouble understanding what "all" refers to.

Maybe use `read_filters` instead? `session.filter.read_filters(role_name)` is a bit redundant, but at least it's clear we read all filters for specified role. It's also consistent with `read_permissions`.

Reply:
we are at filter entity so we reading filters values , many entities already use this function in the same context , why we should rename it here?.

Reply:
Other entities with that function don't require argument here, so there is no ambiguity if "all" refers to entity (filter) or argument (role).

Feel free to disagree and resolve conversation. I don't require this to be changed, I'm only raising concern.

Reply:
if you look at all the functions of this entity:
```python
class FilterEntity(BaseEntity):
    def create(self, role_name, values):

    def search(self, role_name, value):
    
    def read(self, role_name, entity_name, widget_names=None):

    def update(self, role_name, entity_name, values):

    def delete(self, role_name, entity_name):
``` 
why this function has to be an exception?

Reply:
OK, let's keep it your way.